core/lexicon/en/tv.inc.php

```diff
@@ -10,56 +10,56 @@
-$_lang['tv_err_nf'] = 'Template variable not found.';
-$_lang['tv_err_nfs'] = 'Template variable not found with key: [[+id]]';
-$_lang['tv_err_ns'] = 'Template variable not specified.';
-$_lang['tv_err_reserved_name'] = 'A Template Variable cannot have the same name as a Resource field.';
-$_lang['tv_err_save_access_permissions'] = 'An error occured while attempting to save Template Variable access permissions.';
-$_lang['tv_err_save'] = 'An error occurred while saving the Template Variable.';
-$_lang['tv_inuse'] = 'The following document(s) are currently using this Template Variable. To continue with the delete operation click the Delete button otherwise click the Cancel button.';
+$_lang['tv_err_ae'] = 'A TV already exists with the name "[[+name]]".';
+$_lang['tv_err_invalid_name'] = 'You have invalid characters in your TV name.';
+$_lang['tv_err_locked'] = 'TV locked!';
+$_lang['tv_err_nf'] = 'TV not found.';
+$_lang['tv_err_nfs'] = 'TV not found with key: [[+id]]';
+$_lang['tv_err_ns'] = 'TV not specified.';
+$_lang['tv_err_reserved_name'] = 'A TV cannot have the same name as a Resource field.';
+$_lang['tv_err_save_access_permissions'] = 'An error occured while attempting to save TV access permissions.';
+$_lang['tv_err_save'] = 'An error occurred while saving the TV.';
+$_lang['tv_inuse'] = 'The following document(s) are currently using this TV. To continue with the delete operation click the Delete button otherwise click the Cancel button.';
 $_lang['tv_inuse_template'] = 'The following template(s) are currently using this TV: [[+templates]].<br /><br />Please detach the TV from the template(s) before deleting it.';
 $_lang['tv_input_options'] = 'Input Options';
-$_lang['tv_input_options_msg'] = '<p>Here you can edit the input options for the Template Variable, specific to the type of input render that you select.</p>';
-$_lang['tv_lock'] = 'Lock variable for editing';
-$_lang['tv_lock_msg'] = 'Users must have the edit_locked attribute in order to edit this Template Variable.';
-$_lang['tv_management_msg'] = 'Manage additional custom content fields (Template Variables) for your documents.';
-$_lang['tv_msg'] = 'Here you can create/edit Template Variable. Template Variables must be assigned to templates in order to access them from snippets and documents.';
-$_lang['tv_name'] = 'Variable Name';
-$_lang['tv_new'] = 'Create Template Variable';
-$_lang['tv_novars'] = 'No Template Variables found';
+$_lang['tv_input_options_msg'] = '<p>Here you can edit the input options for the TV, specific to the type of input render that you select.</p>';
+$_lang['tv_lock'] = 'Lock TV for editing';
+$_lang['tv_lock_msg'] = 'Users must have the edit_locked attribute in order to edit this TV.';
+$_lang['tv_management_msg'] = 'Manage additional custom TVs for your documents.';
+$_lang['tv_msg'] = 'Here you can create/edit Template Variable (TV). TVs must be assigned to templates in order to access them from snippets and documents.';
```

Comment:
Where possible we should use the [abbreviation element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr):
```suggestion
$_lang['tv_msg'] = 'Here you can create/edit <dfn>Template Variable</dfn> (<abbr title="Template Variable">TV</abbr>). TVs must be assigned to templates in order to access them from snippets and documents.';
```

Reply:
In my opinion, it is sufficient to add the `<abbr>` tag only on this `tv_msg` line, as you suggested.
For the rest of the lines, this is not very useful:
- The familiar TV is already used and users have no questions;
- Bugs may appear, as in https://github.com/modxcms/revolution/issues/14437

Reply:
In the panel descriptions it's useful, furthermore I agree.

Reply:
In the panel descriptions it's useful, furthermore I agree.